deploy/os/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: workspace-controller
+  labels:
+    app: che-workspace-controller
```

Comment:
let's not introduce one more mentioning of che here

Reply:
It should match the other labels, that's why it's there now. I can update all of our labels in this PR I suppose.

Reply:
Done :)